src/libs/masq.js

```diff
@@ -1 +1,112 @@
-module.exports = require('masq-client/dist/masq')
+const Masq = require('masq-lib')
+
+const Error = require('../adapters/error').default
```

Comment:
can you use the new import syntax when it's possible (or maybe it is not)

Reply:
moved the whole file to es6 syntax !